webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -322,14 +323,16 @@ def update_response_parameter_scatterplot(
                         PlotModel(
                             x_axis=x_data.values.flatten(),
                             y_axis=y_data.values.flatten(),
-                            text="Mean",
-                            name=f"{repr(ensemble)}: {active_response}x{active_parameter}@{response.axis[x_index]}",
+                            text="Mean",  # TODO ask about this!
```

Comment:
`# TODO ask about this!` what is this comment about?

Reply:
XD that's hilarious, maybe they put it to have something? my comment is indeed for finding out why this is called mean, and i also thought that it didn't make sense, but since i don't know much about the domain or statistics, i thought i'd ask ^^

ok i figured out that no plot was drawn because the plot model wants to get a name. extending the plot model to accept no given name, we fix the no plot and circumvent having empty space / an empty line in the tooltip. yeahy : )